torch/extension.h

```diff
@@ -1,6 +1,9 @@
 #pragma once
 
+#ifndef INDUCTOR_CPP_WRAPPER
```

Comment:
Suggest to use `TORCH_INDUCTOR_CPP_WRAPPER` per suggested by @ezyang

Reply:
Done